Map-editor documentation (#3341)
workadventure/workadventure: docs/maps/editor/property/jitsi.md

## Jitsi meeting property
On your map, you can define special zones where a Jitsi meeting will be opened when a user enters the area.

> YOu can also define an entity that will be trigger a Jitsi meeting room.

Author: ```suggestion
You can also trigger a Jitsi meeting room from an entity menu.
```
